src/app.rs

```diff
@@ -363,70 +336,6 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
         }
     }
 
-    /// Bootstrap the dom with the vdom by taking over all children of the mount point and
-    /// replacing them with the vdom if requested. Will otherwise ignore the original children of
-    /// the mount point.
-    fn bootstrap_vdom(&self, mount_type: MountType) -> El<Ms> {
```

Comment:
This is directly relevant to issue #277 

From the code that I was working with, `mount_type` was only ever passed in with `::Takeover` once `App::new()` was made private. Given that this case only ever returns the result of `El::empty(Tag::Placeholder)`, I opted to replace calls to this function with that.